lib/rubocop/cop/fablicop/unexpected_identifiers.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Fablicop
+      class UnexpectedIdentifiers < Base
+        MSG = 'Check if @current_user exists.'
+
+        def on_ivar(node)
+          identifiers = cop_config['Identifiers']
+          return unless identifiers && identifiers.is_a?(Array) && cop_config['Identifiers'].include?(node.source)
```

Comment:
Can't we use `identifiers` here?

```suggestion
          return unless identifiers && identifiers.is_a?(Array) && identifiers.include?(node.source)
```